### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]: Keyboard focus is not visible after deleting the keys which is present under key for web app properties window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1457]
**Repro Steps:**

1. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
2. Navigate to and select "Show properties" menu item .
3. Properties window will open, then tab to "Add" button and hit enter.
4. Enter any key details and press enter then tab to delete button and hit enter then observe the keyboard focus

**Expected Result:**
Keyboard focus should be visible with in the table which is present under key header  after deleting the key 

**Actual Result:**
Keyboard focus is not visible after deleting the key and for the next tab keyboard focus is in "Add" button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![keyboard focus is not visible after deleting key in show properties window](https://user-images.githubusercontent.com/22956130/36020695-8b032a60-0da9-11e8-92aa-c2b7e3f16525.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by jdneo** [comment]
Fixed

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0 . Issue got fixed .
Please add closed tag 
![1457-fixed](https://user-images.githubusercontent.com/22956130/39757864-bba6fe9e-52eb-11e8-9ef0-d3208c83c327.jpg)
to it.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749883
